bundles/org.openhab.binding.astro/README.md

```diff
@@ -1,6 +1,7 @@
 # Astro Binding
 
 The Astro binding is used for calculating 
+
```

Comment:
To render as actual dots the indentation before the `*` should be removed. The newline only makes the code textblock show as code textblock and therefor shows a separate lines.

Reply:
My bad...looked good in my editor but you're right. Fixing.